src/odemis/dataio/test/multi_tiff_test.py

```diff
@@ -0,0 +1,101 @@
+
```

Comment:
Please add the standard boilerplate for the test cases. It should have the shebang (as it's supposed to be executable by itself) + the copyright notice.